From 290c32fb19d901306c4da1119a679112490cb838 Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Mon, 15 Aug 2011 21:29:51 -0700
Subject: [PATCH:luit] Remove check for zlib in configure script

If libfontenc needs it, it will either be linked with it or
provide -lz in its pkg-config library requirements.

Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
Reviewed-by: Dan Nicholson <dbn.lists@gmail.com>
---
 configure.ac |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index a36aa94..c3cd947 100644
--- a/configure.ac
+++ b/configure.ac
@@ -40,8 +40,6 @@ AC_CANONICAL_HOST
 AC_CHECK_HEADERS([pty.h stropts.h sys/param.h sys/select.h])
 AC_CHECK_FUNCS([select grantpt])
 
-AC_CHECK_LIB(z, gzopen,, AC_MSG_ERROR([zlib not found]))
-
 AC_ARG_WITH(localealiasfile,
         AS_HELP_STRING([--with-localealiasfile=<path>],
                        [The locale alias file (default: ${datadir}/X11/locale/locale.alias)]),
-- 
1.7.3.2

